workspaces/workspace-templates/code-completion.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: "Workspace template code completion"
-description: "Learn how to use code completion when creating workspace templates."
+description:
+  "Learn how to use code completion when creating workspace templates."
```

Comment:
I think having this on two lines breaks the build

Reply:
Prettier did this, not me. Hmm I'll check it out - currently running build based off of the suggestions before this

Reply:
Yeah, Prettier will do that so I just manually format all but the metadata (I'm sure there's a better way, but that's how I've been doing it :))

Reply:
@khorne3 - everything on the preview seems fine 😕 

![preview-1](https://user-images.githubusercontent.com/11184711/124666463-16841d00-de7c-11eb-9572-d885c76a9710.JPG)

Reply:
I stand corrected! Good to see this won't be a problem anymore

Reply:
Yea me too. If it ended up causing a problem, I'd argue that it's something we'd have to consider filing upstream to prettier or addressing in our prettier config. Otherwise it implies that a code formatter (strictly formatting) is having an unintended output consequence. That kind of defeats the purpose of what prettier is supposed to be doing, though I imagine there are few edge cases where this has happened (ex: `<pre>` tags).

Great stuff though - hopefully this means everyone can just enable the prettier VS Code extension and trust `ctrl + save` :p - afterall, that's all I did!